lib/bunch/list.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Bunch.List do
+  @moduledoc """
+  A bunch of helper functions for list manipulation.
+  """
+
+  @doc """
+  Generates a list basing on the given `seed`. Allows to return the final seed.
```

Comment:
I don't think it should be called seed, as you do not return `final seed` - you return the last result created out of seed (that could've been the seed for another call). I would rather focus more on the function:
```suggestion
  Generates a list using generator function and provided inital accumulator. Returns the final accumulator.
```
(and update the rest of the docs accordingly)